config/v3_4_experimental/types/config.go

```diff
@@ -25,3 +31,50 @@ var (
 		PreRelease: "experimental",
 	}
 )
+
+func (cfg Config) Validate(c path.ContextPath) (r report.Report) {
+	systemdPath := "/etc/systemd/system/"
+	units := make(map[string]string)
```

Comment:
Nit: this is integral with the loop that populates it, so let's drop the blank line between them.